### PR TITLE
customize the generate sourcemap file or not

### PR DESCRIPTION
**Pull request by pengcao1**
Generate android-sourcemap file is not needed under may times
but, It's run the script to generate map file at every release build
 
-  QA will contain Debug/Release
-  UAT will contain Debug/Release
-   Prod will contain Debug/Release

In most cases, only the Production Release version need to generate the sourcemap file

### CODE REVIEW

**Comment by pengcao1** [comment]
Could anybody help to review this PR ?

**Comment by HeshamMegid** [comment]
@pengcao1 thanks for the PR! We're testing this and we'll be merging it soon.

File: android/build.gradle [hunk 1]
```diff
@@ -50,7 +50,8 @@ task upload_sourcemap(type: Exec) {
 }
 
 tasks.whenTaskAdded { task ->
-    if (task.name == 'preReleaseBuild') {
+    if (task.name == 'preReleaseBuild' &&
+        rootProject.hasProperty("instabugUploadEnable") ? rootProject.instabugUploadEnable : false) {
```
Comment (alyezz) [inline android/build.gradle]:
```suggestion
        rootProject.hasProperty("instabugUploadEnable") ? rootProject.instabugUploadEnable : true) {
```

Comment (alyezz) [reply android/build.gradle]: If the property is not defined, then the upload should proceed.

**Pull request merged**